Documentation/design-docs/code-versioning-profiler-breaking-changes.md

```diff
@@ -0,0 +1,29 @@
+# Code Versioning Profiler Breaking Changes #
```

Comment:
> So you saying that code map is broken after tiered re jit?
Yes, the end to end workflow would be broken. GetILToNativeMapping will give you a valid IL to Native map for the first code body which is jitted. However if the same method JITs a second time, the second code body may have a different native to il mapping than the first. The current API provides you no mechanism to access  the native to il map that corresponds to that second code body.